gamestonk_terminal/cryptocurrency/coinpaprika/coinpaprika_controller.py

```diff
@@ -0,0 +1,310 @@
+"""CoinPaprika Controller"""
+__docformat__ = "numpy"
+# pylint: disable=R0904, C0302, W0622
```

Comment:
If possible I would change the magic numbers to the string statement, e.g. `# pylint: too-many-branches`.

This allows that the dev understands straight what's going on rather than having a magic number displayed.

Reply:
I changed it but for `too-many-lines` tests were failing on anaconda environment, dunno why ❓

Reply:
Not important tho! This was a nitpicky :)